docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
+We've introduced 2 new columns in the [project list view](https://pipedream.com/projects): **Owner** and **Access**.
+
+**Owner** indicates who within the workspace owns each project. This is typically the person who created the project.
+
+![Project Listing (Owner)](./images/project-listing-owner.png)
+
+::: warning Some projects may not have owners
+Projects created before February 2024 don't automatically have owners.
+:::
+
+**Access** indicates who within the workspace has access to each project, and this can be displayed as "me", "Workspace", or "N members".
+
+![Project Listing (Access)](./images/project-listing-access.png)
+
+### Permissions
+For workspace members who have access to the project (either because it's shared with the entire workspace or with that member specifically), below are the permissions for the project.
```

Comment:
```suggestion
Workspace owners and admins are able to perform all actions in projects, whereas members are restricted from performing certain actions.
```

Reply:
Just rephrasing to introduce the idea of permissions, then letting the table below speak for itself.